test/cpp/nativert/test_mpmc_queue.cpp

```diff
@@ -53,6 +53,7 @@ TEST(MPMCQueueTest, ConcurrentAccess) {
       int out = 0;
       while (!queue.readIfNotEmpty(out)) {
         // Wait until an element is available
+        std::this_thread::yield();
```

Comment:
Does the queue have a method to wait until read available? Use spinning wait is quite strange.

Reply:
Yeah but we don't provide a blocking wait for now. Internally we use a semaphore paired with this (not upstreamed yet)